packages/my-extended-retail-app/package.json

```diff
@@ -1,102 +1,102 @@
 {
-    "name": "my-extended-retail-app",
-    "version": "2.8.0-dev",
-    "license": "See license in LICENSE",
-    "engines": {
-        "node": "^16.0.0 || ^18.0.0",
-        "npm": "^7.0.0 || ^8.0.0 || ^9.0.0"
+  "name": "my-extended-retail-app",
+  "version": "2.8.0-dev",
+  "license": "See license in LICENSE",
+  "engines": {
+    "node": "^16.0.0 || ^18.0.0",
+    "npm": "^7.0.0 || ^8.0.0 || ^9.0.0"
+  },
+  "ccExtensibility": {
+    "extends": "retail-react-app",
+    "overridesDir": "pwa-kit-overrides"
+  },
+  "devDependencies": {
+    "@chakra-ui/icons": "^1.1.7",
+    "@chakra-ui/react": "^1.8.9",
+    "@chakra-ui/skip-nav": "^1.2.6",
+    "@chakra-ui/system": "^1.12.1",
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
+    "@formatjs/cli": "^6.0.4",
+    "@lhci/cli": "^0.11.0",
+    "@loadable/component": "^5.15.3",
+    "@peculiar/webcrypto": "^1.4.2",
+    "@tanstack/react-query": "^4.0.10",
+    "@tanstack/react-query-devtools": "^4.29.1",
+    "@testing-library/dom": "^9.0.1",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/react-hooks": "^7.0.2",
+    "@testing-library/user-event": "^12.8.3",
+    "base64-arraybuffer": "^0.2.0",
+    "bundlesize2": "^0.0.31",
+    "card-validator": "^8.1.1",
+    "commerce-sdk-react-preview": "^2.8.0-dev",
+    "cross-env": "^5.2.1",
+    "cross-fetch": "^3.1.5",
+    "focus-visible": "^5.2.0",
+    "framer-motion": "^3.10.6",
+    "full-icu": "^1.5.0",
+    "helmet": "^4.6.0",
+    "jest-fetch-mock": "^2.1.2",
+    "js-cookie": "^3.0.1",
+    "jsonwebtoken": "^9.0.0",
+    "jwt-decode": "^3.1.2",
+    "lodash": "^4.17.21",
+    "msw": "^1.2.1",
+    "nanoid": "^3.3.4",
+    "njwt": "^1.2.0",
+    "prop-types": "^15.8.1",
+    "pwa-kit-dev": "2.8.0-dev",
+    "pwa-kit-react-sdk": "2.8.0-dev",
+    "pwa-kit-runtime": "2.8.0-dev",
+    "query-string": "^7.1.3",
+    "raf": "^3.4.1",
+    "randomstring": "^1.2.3",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-helmet": "^6.1.0",
+    "react-hook-form": "^6.15.8",
+    "react-intl": "^5.25.1",
+    "react-router-dom": "^5.3.4",
+    "retail-react-app": "2.8.0-dev"
+  },
+  "peerDependencies": {
+    "@chakra-ui/system": "^1.12.1"
+  },
```

Comment:
this probably should be moved to devDependencies?

Reply:
No, this was already this way (unchanged in this PR) and it was part of a whole tangled problem @adamraya fixed before

Reply:
The diff here is misleading, I just fixed my bad indentation

Reply:
@adamraya i'm just curious what's the reason for making `@chakra-ui/system` as a peer dependency? If this is done by mistake, i'm happy to create a PR to fix it.

Reply:
The change was required in v2 to force NPM to use chakra v1.

PR: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1012


Issue on Chakra repo: "peerDependencies": {
    "@chakra-ui/system": ">=1.0.0",

Reply:
So this is something we can drop in `v3` it sounds like. But it also has nothing to do with this PR